www/pages/api.php

```diff
@@ -79,18 +80,27 @@
 if ($uid != '') {
 	$page->users->updateApiAccessed($uid);
 	$apiRequests = $page->users->getApiRequests($uid);
-	if ($apiRequests > $maxRequests) {
-		showApiError(500, 'Request limit reached (' . $apiRequests . '/' . $maxRequests . ')');
+	if ($apiRequests['num'] > $maxRequests) {
```

Comment:
This doesn't work, the method doesn't return a array. https://github.com/nZEDb/nZEDb/blob/dev/nzedb/Users.php#L1345